### PR TITLE
Added thomus hainii

### PR DESCRIPTION
**Pull request by mjukus**

### CODE REVIEW

**Comment by mjukus** [comment]
I'd like to add a special bird of mine

File: params/jack_bird.R [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 4,67
```
Comment (twinkarma) [inline params/jack_bird.R]: Please use a dot instead of comma
```suggestion
sig2 <- 4.67
```

**Review by twinkarma — CHANGES_REQUESTED** [review]
See suggestion!

**Review by twinkarma — APPROVED** [review]
Thanks for your contribution

**Pull request merged**